docs/guide/tooltip.md

```
@@ -84,7 +84,7 @@ We can make the text bigger too:
 
 !!! tip
 
-      AI is crazy good at CSS! Just describe what you want and it'll be able to do it for sure.
+      AI tools are very good at CSS. Describe what you want, and they can help generate it.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)